firmware/boron-ins-fsm/src/consoleFunctions.h

```diff
@@ -1,22 +1,21 @@
 /*
  * Brave firmware state machine for single Boron
  * written by Heidi Fedorak, Apr 2021
-*/
+ */
 
 #ifndef CONSOLEFUNCTIONS_H
 #define CONSOLEFUNCTIONS_H
 
 //*************************macro defines**********************************
 
-
 //******************global variable declarations*******************
 
 //*************************function declarations*************************
 
-//setup() functions 
+// setup() functions
```

Comment:
Do you know which clang parameter adds this space after the `//`? Because when I run it with the current config, it doesn't seem to add it. But I do like this space, so it would be nice if it were in there.

Reply:
It seems that unfortunately, this parameter does not exist until version 13 of clang-format, but version 12 is the newest version supported for Ubuntu Xenial. I think the spaces came from me using a newer version. Do you think it is worthwhile to setup a newer .clang-format which can be run locally, which also conforms with the version 12 rules?

Reply:
Ah well. No, I don't think that's worth it. You having run it once to add the spaces seems good to me. It's just another good reason to upgrade our Ubuntu. I don't think I'm ready for that today though, lol. Thanks though!